src/accounts/AccountTabs.tsx

```diff
@@ -35,7 +37,7 @@ const AccountTabs: FC<Props> = ({activeTab}) => {
     },
     {
       id: 'organizations',
-      enabled: isFlagEnabled('createDeleteOrgs'),
+      enabled: CLOUD,
       link: `/orgs/${orgID}/accounts/orglist`,
```

Comment:
The organization tab cannot be accessed from OSS, where there is no multi-org or access to quartz APIs.